### PR TITLE
Add owners file to cassandra example

### PR DESCRIPTION
**Pull request by aledbf**

### CODE REVIEW

File: cassandra/OWNERS [hunk 1]
```diff
@@ -0,0 +1,6 @@
+reviewers:
+  - ahmetb
```
Comment (ahmetb) [inline cassandra/OWNERS]: definitely don't add me :) I don't know anything about Cassandra and trying to reduce my duties on this repo. :)

Comment (aledbf) [reply cassandra/OWNERS]: done :)

**Pull request merged**